api-gateway/src/main/java/com/petclinic/bffapigateway/exceptions/GenericHttpException.java

```diff
@@ -12,6 +13,7 @@
  */
 
 @Data
+@EqualsAndHashCode(callSuper = false)
```

Comment:
What is this doing?

Reply:
This  to stop the @Data to calling the weird super constructor. This is a funny quirk of lomdok !